src/content/blog/astro-tsc-2025.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: '2025 Technical Steering Committee'
+description: 'Meet Astro’s 2025 Technical Steering Committee and learn how our open governance model supports the framework’s growth, performance, and community-led development.'
+publishDate: '2025-04-18'
+authors:
+  - fred
+coverImage: '/src/content/blog/_images/astro-tsc-2025/blog-hero.jpg'
+socialImage: '/src/content/blog/_images/astro-tsc-2025/og-image.jpg'
+lang: 'en'
+---
+
+import Mention from "/src/components/Mention.astro"
+
+The Astro project has a new Technical Steering Committee (TSC) for 2025! I am excited to announce <Mention name="ema" /> <Mention name="florian" /> and <Mention name="matt" /> are now part of the Astro TSC, joining Matthew Phillips who will remain on the TSC. 
+
+In addition, I am also proud to recognize <Mention name="matt" /> as the official Astro framework team lead going forward, joining <Mention name="sarah" /> who will continue as the project's official docs team lead.
+
+This announcement is part of our ongoing commitment to open governance and bringing you the best framework for building content-driven websites. If you're curious to learn more, you can read our entire set of public governance docs [on GitHub](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#get-involved), see our [full list of maintainers](https://github.com/withastro/.github/blob/main/MEMBERS.md), and chat with us [on Discord](https://astro.build/chat).
```

Comment:
```suggestion
This announcement is part of our ongoing commitment to open governance and bringing you the best framework for building content-driven websites. If you're curious to learn more, you can read our entire set of public [governance docs on GitHub](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#get-involved), see our [full list of maintainers](https://github.com/withastro/.github/blob/main/MEMBERS.md), and chat with us [on Discord](https://astro.build/chat).
```
This might have gotten lost in the rewrite, but I still suggest making the governance docs part of the linked text! (It will stand out more, and make it clear it's not just the repo for people who are skimming)